content/en/continuous_delivery/features/changes_detection.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Changes Detection
```

Comment:
(will need to be changed in multiple places)

I think `Changes Detection` is too hard to interpret. I recommend to align it with our UI: `Code Changes`.